src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/osd-summary.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import _ from 'lodash';
+
+@Pipe({
+  name: 'osdSummary'
+})
+export class OsdSummaryPipe implements PipeTransform {
+  transform(value: any): any {
```

Comment:
Please let's not do this. We should clearly define the contract/interface in our functions. What's the input type supported by this pie, and what's the output of it.